We build python3 without gettext support built-in, so these function calls _will_
fail.

diff --git a/passwordsafe.in b/passwordsafe.in
index b66e0c7..c68724d 100755
--- a/passwordsafe.in
+++ b/passwordsafe.in
@@ -31,8 +31,6 @@ def install_excepthook():
 if __name__ == "__main__":
     install_excepthook()
 
-    locale.bindtextdomain('passwordsafe', localedir)
-    locale.textdomain('passwordsafe')
     gettext.bindtextdomain('passwordsafe', localedir)
     gettext.textdomain('passwordsafe')
 
